src/basic_recipes/streamplot.jl

```diff
@@ -213,10 +213,18 @@ function plot!(p::StreamPlot)
         rotations = map(x -> x[2], data)
     end
 
+    if p.arrow_size[] === automatic
+        if N == 3
+            arrow_size = 0.2 * minimum(p.limits[].widths) / minimum(p.gridsize[])
+        else
+            arrow_size = 15
+        end
+    end
```

Comment:
This is missing an else branch which is why tests are failing. Might also be good to reorganize this so that `p.arrow_size` can change from automatic to some value. I.e. use `arrow_size = map(p.arrow_size, p.limits, p.gridsize) do ...`.